src/main.rs

```diff
@@ -79,19 +88,47 @@ fn symbolic_revert(sub_matches: &ArgMatches) -> Result<(), String> {
         .parse()
         .unwrap();
 
-    let query = yools::encoder::encode_revert_unreachable::<EVMInstructions>(&ast, loop_unroll);
+    let eval_strings = if let Some(eval) = sub_matches.value_of("eval") {
+        eval.split('|')
+            .map(|e| e.to_string())
+            .collect::<Vec<String>>()
+    } else {
+        vec![]
```

Comment:
imo `Option` would be nicer than empty vec

Reply:
But then you also have `Some(empty vec)`?

Reply:
Only if the user passed an empty eval vec?

Reply:
I'm just saying it's a redundancy - you can get the same behaviour both by passing `None` and by passing `Some(!vec[])`